src/audio/buffer.c

```diff
@@ -19,6 +19,10 @@
 #include <stddef.h>
 #include <stdint.h>
 
+#ifdef __ZEPHYR__
```

Comment:
Can we remove the #ifdef here and just have and always declare this in each file (using an empty macro definition with no impact if xtos is used). This will simplify and make the code cleaner.